packages/py-api/py_api/controllers/getting_logs_controller.py

```diff
@@ -0,0 +1,44 @@
+import pathlib

```

Comment:
file should be renamed to logs_controller

Reply:
Done